handlers/search.js

```diff
@@ -64,7 +57,7 @@ function getPagination(url, currentPage, perPage, maxPages, totalCount) {
 function getFilters(query) {
   const filters = {};
   const ensureIsArrayAndQuoteElements = (values) => (
-    ensureIsArray(values).map((val) => `"${val}"`)
+    values.map((val) => `"${val}"`)
```

Comment:
I haven't tested the code, but it seems that you're removing `ensureIsArray` because Joi converts single elements to array, right? If that's the case, the name of this function is now incorrect: it's just quoting elements, and not ensuring it's an array. Please, rename it accordingly (maybe `quoteElements()`))